webapp/packages/core-connections/src/ConnectionExecutionContext/ConnectionExecutionContext.ts

```diff
@@ -163,3 +170,11 @@ export class ConnectionExecutionContext implements IConnectionExecutionContext {
     return callback(this.context);
   }
 }
+
+function mapAsyncTaskInfo(info: AsyncTaskInfo): IConnectionExecutionContextUpdateTaskInfo {
+  return {
+    success: info.status === ASYNC_TASK_STATUS_FINISHED && !info.error,
```

Comment:
you don't need to check `info.status` task will throw an exception on failure

Reply:
check: 
https://github.com/dbeaver/cloudbeaver/blob/eae35ca957a9cd6758a166fffc930db4a3d13c42/webapp/packages/core-sdk/src/AsyncTask/AsyncTask.ts#L118-L129